react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,49 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.hidden {
+  display: none;
+}
+
+#root {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+label {
+  font-size: 1.5rem;
+  font-weight: bold;
+}
+
+input {
+  padding: 0.1rem 0;
+  font-size: 1.5rem;
+}
+
+.far.fa-times-circle {
+  position: relative;
+  top: 0.2rem;
+  left: 0.6rem;
+  color: red;
+  font-size: 2rem;
+}
+
+.far.fa-check-circle {
+  position: relative;
+  top: 0.2rem;
+  left: 0.6rem;
+  color: green;
+  font-size: 2rem;
+}
```

Comment:
You don't want to directly change classes from a css library. You are changing the default behavior of the class which can cause issues in other places in your app and cause confusion with other people working on the project. It is best to just make your own class and use that on the element you are trying to effect in conjunction with the class from the css library.

Reply:
In your CSS use a custom class like:

```CSS
.icon {
  position: relative;
  top: 0.2rem;
  left: 0.6rem;
  color: red;
  font-size: 2rem;
}

.icon-red {
  color: red;
}

.icon-green {
  color: green;
}
```